tests/unit/test_asynchronous.py

```diff
@@ -185,7 +185,7 @@ async def test_forget():
     _get_fallback_session.cache_clear()
 
     if os.path.exists(settings.PRIVATE_KEY_FILE):
-        os.remove(settings.PRIVATE_KEY_FILE)
+        delete_private_key_file()
```

Comment:
What happens if I run pytest on my PC ? Would I then lose my private key ?
I know this PR does not create that issue, but we should solve it.

Reply:
in pytest the private key name now has a suffix "test"